packages/rollup-core/src/types/data/l2-data-service.ts

```diff
@@ -9,14 +9,14 @@ export interface L2DataService {
    * @param transaction The transaction to insert.
    * @throws An error if there is a DB error.
    */
-  insertL2Transaction(transaction: TransactionAndRoot): Promise<void>
+  insertL2TransactionOutput(transaction: TransactionOutput): Promise<void>
 
   /**
    * Builds an L2-only batch if there are unbatched L2 Transactions with different timestamps.
    *
    * @returns The number of the L2 Batch that was built, or -1 if one wasn't built.
    */
-  tryBuildL2OnlyBatch(): Promise<number>
+  tryBuildOccBatchNotPresentOnL1(): Promise<number>
```

Comment:
Could be good to update this and the next method doc to reflect new naming decisions, even if just by adding the qualifier that this is within the context of the sequencer/an OCC.

Reply:
Good call -- I hadn't don't a blanket search on `batch`. Will do!